src/EditorFeatures/CSharpTest/Diagnostics/DiagnosticAnalyzerDriver/DiagnosticAnalyzerDriverTests.cs

```diff
@@ -39,7 +39,15 @@ public class DiagnosticAnalyzerDriverTests
                 await DiagnosticProviderTestUtilities.GetAllDiagnosticsAsync(analyzer, document, new Text.TextSpan(0, document.GetTextAsync().Result.Length));
                 analyzer.VerifyAllAnalyzerMembersWereCalled();
                 analyzer.VerifyAnalyzeSymbolCalledForAllSymbolKinds();
-                analyzer.VerifyAnalyzeNodeCalledForAllSyntaxKinds(new HashSet<SyntaxKind>());
+
+                var syntaxKinds = new HashSet<SyntaxKind>();
+
+                // PROTOTYPE: investigate whether this should be added or not?
```

Comment:
💭 I have no idea why this would be needed (i.e. what makes the new kinds special when "nothing" before was)

Reply:
no idea :( need to investigate history of this test later.

---
In reply to: [185127444](https://github.com/dotnet/roslyn/pull/26405#discussion_r185127444) [](ancestors = 185127444)